Lib/test/test_typing.py

```diff
@@ -2778,6 +2778,15 @@ class HasForeignBaseClass(mod_generics_cache.A):
 
 gth = get_type_hints
 
+class ForRefExample():
```

Comment:
No parentheses here, and 2 blank lines between top-level definitions:

```suggestion

class ForRefExample:
```